.github/workflows/build.yml

```diff
@@ -76,25 +76,16 @@ jobs:
         with:
           python-version: 3.9
       # This step ensures that everything is installed
-      - name: Install code cleaning components
+      - name: Upgrade pip
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -e '.[dev]'
-
-      - name: Lint code
-        uses: psf/black@stable
 
-      - name: Flake code
-        run: |
-          python -m flake8 src tests
-
-      - name: Check import ordering
-        uses: isort/isort-action@master
-        with:
-          configuration: --check-only
+      - name: Run Ruff
+        uses: astral-sh/ruff-action@v1
```

Comment:
Just curious about where you go to find these GitHub action use strings.

Reply:
Google searched "GitHub Action for X"